### DISCUSSION TITLE
Use own s3 in hosted?

### DISCUSSION THREAD

**Discussion by RickCogley** [outline/outline#1733] [Self Hosting] [answered]
Is it possible to use one’s own S3 in the hosted version?

**Comment by tommoor** [comment, marked as answer]
Hi Rick, I'm afraid it's not possible at this time.

**Reply by RickCogley** [reply]
thanks @tommoor

**Comment by gsarristz** [comment]
Hello, 

Is this still the case? Can we use AWS S3 buckets in the hosted version or do we have to stick with minio? I have trying to get Outline to work with AWS S3 bucket with not much luck so far. 

Thanks,
George

**Reply by tommoor** [reply]
"hosted" in this thread refers to cloud hosted at `getoutline.com`. For self-hosted you can use S3, yes.

**Reply by Abdulvoris-Urolov** [reply]
Image cannot be uploaded to s3. What is the main reason for this? It returns a 400 error
```
<Error
>
<Code
>
InvalidArgument
</Code
>
<Message
>
the region 'us-east-1' is wrong; expecting 'eu-west-1'
</Message
>
<ArgumentName
>
X-Amz-Credential
</ArgumentName
>
<ArgumentValue
>
AKIN7RP4ZM47/20230512/us-east-1/s3/aws4_request
</ArgumentValue
>
<Region
>
eu-west-1
</Region
>
<RequestId
>
3S8F9
</RequestId
>
<HostId
>
Z1JwzHhcgoZgUp27dcpWtSbYl/Z9A=
</HostId
>
</Error>
```
